.eslintrc

```diff
@@ -12,6 +12,9 @@
       "ignoreUrls": true,
       "ignoreComments": false
     }],
+    "react/require-default-props": [0],
```

Comment:
while i dont have an opinion, maybe linting rule changes should happen in separate pr's so their impact could be discussed?

Reply:
https://github.com/ManageIQ/react-ui-components/pull/6